pkg/ami/api.go

```diff
@@ -54,6 +54,9 @@ func Use(ec2api ec2iface.EC2API, ng *api.NodeGroupBase) error {
 	case "ebs":
 		if !api.IsSetAndNonEmptyString(ng.VolumeName) {
 			ng.VolumeName = image.RootDeviceName
+			if ng.AMIFamily == api.NodeImageFamilyBottlerocket {
+				ng.VolumeName = aws.String("/dev/xvdb")
```

Comment:
is there a tidier way of setting this? feels like a bit of a "magic string" (if that is not already a thing